lib/i18n/tasks/missing_keys.rb

```diff
@@ -154,5 +150,32 @@ def collapse_same_key_in_locales!(forest)
       end
       forest
     end
+
+    private
+
+    def plural_keys_for_locale(locale)
+      configuration = load_rails_i18n_pluralization!(locale)
+      if configuration[locale.to_sym].nil?
+        alternate_locale = alternate_locale_from(locale)
+        return Set.new if configuration[alternate_locale.to_sym].nil?
+
+        return set_from_rails_i18n_pluralization(configuration, alternate_locale)
+      end
+      set_from_rails_i18n_pluralization(configuration, locale)
+    rescue SystemCallError, IOError
+      Set.new
+    end
+
+    def alternate_locale_from(locale)
+      re = /(\w{2})-*(\w{2})*/
```

Comment:
The second group can be 3 letters, e.g. https://github.com/svenfuchs/rails-i18n/blob/254b1b43730cde87a727b83b569c08fc59b7e3fa/rails/locale/zh-YUE.yml